fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/BulkDataConstants.java

```diff
@@ -23,6 +23,13 @@
 
     // Import
     public static final String INPUT_FORMAT = MEDIA_TYPE_ND_JSON;
+    public static final List<String> INPUT_FORMATS = Collections.unmodifiableList(Arrays.asList(INPUT_FORMAT));
+    public static final List<String> STORAGE_TYPES =
+            Collections
```

Comment:
the wrapping seems strange to me, I'm not sure what's the correct pattern, but we do need a unified pattern for this.

Reply:
it's at 120 Characters. and based on the tab stop